Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -77,3 +77,9 @@
 
 #string STR_FAN_CURVE_SILENT        #language en-US "Silent"
 #string STR_FAN_CURVE_PERFORMANCE   #language en-US "Performance"
+
+#string STR_DMA_BOOT_ENABLE_PROMPT      #language en-US "DMA Protection"
+#string STR_DMA_BOOT_ENABLE_HELP        #language en-US "Enables DMA protection"
+
+#string STR_IOMMU_HANDOFF_ENABLE_PROMPT      #language en-US "IOMMU handoff at ExitBootServices"

```

Comment:
I would make it a list similar to ME disable option. If we do not do handoff at EBS, when does it exactly happen?

Reply:
I've made it like `UsbStack` instead since it was easier, now the `iommu handoff` doesn't appear in options when DMA is disabled, I hope this is fine. 
As for the handoff, I've updated the description. From what I've found, if we don't do it at EBS, it happens at the start of BDS (Boot Device Selection) in edk2 [boot sequence](https://edk2-docs.gitbook.io/edk-ii-build-specification/2_design_discussion/23_boot_sequence). Haven't found the exact place in code yet, but as for now I think the distinction should be clear.

Reply:
Easier != better. User experience is the most important. I asked t change the handoff option to be a list like the ME disable option. Please apply.

Reply:
Applied, now we have a `oneof` gui element like ME.